specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/associatedTenantsCreateOrUpdate.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "api-version": "2023-04-01",
+    "billingAccountName": "00000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+    "associatedTenantName": "11111111-1111-1111-1111-111111111111",
+    "parameters": {
+      "properties": {
+        "friendlyName": "Contoso Finance",
```

Comment:
I think we should explore having a mechanism where tenant names are synced, similar to what we do for CSP. I suspect things will start getting confusing for customers when tenant names start changing, and people fail to update these associations with updated names. I think the SLA on the sync  could be low (e.g. 24 hours).

Reply:
We explored this in our GA scrum today, and Ram and Amber are both opposed to the behavior for a few reasons. 
- First, only ProvisioningTenant as an "approval" process. There is no such process for BillingManagementTenant.
- Second, many field led customers never change their directory name from "Default" which could create confusion when all of the "AssociatedTenants" have the same friendly name
- Third, Ram is wanting to move away from CommerceCache supporting the project/customer tenant name sync that is existing. This should be a responsibility of the relationship service.

Reply:
#1: Why does the type of relationship matter? Don't we want to describe the name for *every* association so that anyone with access to a billing account can see the full set of tenants associated with it?

#2: You could use the domain name which I believe must be unique.

#3: Fully agreed for CSP it should be the relationship service managing this. In this case associatedTenants *is* the relationship service. :)

Curious if @grzegorzzygmunt has an opinion here as well.

Reply:
I don't see enough value in syncing names with AAD. We have had ~160 customers so far and none of the customers have complained about display name.  It is more work, and we will also open ourselves to the CRIs/support cases because we'll always have latency if we try to sync the name with AAD. We should keep display name for now and if customers ask for it, we can add AAD tenant name as a new property later (We would need two properties as a lot of customers never update their tenant's name).

Reply:
Fair enough... Can we at least call it "displayName" for now?

Reply:
Yes, I am ok with "displayName", Braden?

Reply:
Updated